src/features/main/Main.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+
+import Button from "../../components/Button";
+
+function Main() {
+
+  return (
+    <h1>This is memoRoom Main</h1>
+  )
```

Comment:
```suggestion
  );
```
세미콜론이 빠져 있네요 ㅎㅎ 저희 린트 설정및 허스키가 설정이 되어 있는 부분 한번 확인 하시면 좋을것 같아요!!